scalafix-core/src/main/scala/scalafix/rewrite/ExplicitReturnTypes.scala

```diff
@@ -20,18 +22,39 @@ case class ExplicitReturnTypes(mirror: Mirror)
 
   def defnName(defn: Defn): Option[Name] = Option(defn).collect {
     case Defn.Val(_, Seq(Pat.Var.Term(name)), _, _) => name
+    case Defn.Var(_, Seq(Pat.Var.Term(name)), _, _) => name
     case Defn.Def(_, name, _, _, _, _) => name
   }
 
+  def visibility(mods: Traversable[Mod]): MemberVisibility =
+    mods
+      .collect {
+        case _: Mod.Private => MemberVisibility.Private
+        case _: Mod.Protected => MemberVisibility.Protected
+      }
+      .headOption
+      .getOrElse(MemberVisibility.Public)
+
+  def kind(defn: Defn): Option[MemberKind] = Option(defn).collect {
+    case _: Defn.Val => MemberKind.Val
+    case _: Defn.Def => MemberKind.Def
+    case _: Defn.Var => MemberKind.Var
```

Comment:
It seems we should make the `implicit` part configurable. I propose we include it under MemberKind, so that 
`explicitReturnTypes.memberKind = [Implicit]` means "insert explicit types for any implicit member, regardless if it's def/var/val. WDYT/